docs/guides/update-guide/connectors/010-to-020.md

```diff
@@ -0,0 +1,173 @@
+---
+id: 010-to-020
+title: Update 0.1 to 0.2
+description: "Review which adjustments must be made to migrate from Connector SDK 0.1.x to 0.2.0."
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+<span class="badge badge--primary">Intermediate</span>
+
+The following sections explain which adjustments must be made to migrate from
+[Connector SDK](/components/integration-framework/connectors/custom-built-connectors/connector-sdk.md)
+0.1.x to 0.2.0.
+
+:::caution
+
+Be aware that the update from 0.1 to 0.2 is not backward-compatible.
```

Comment:
Personally, instead of writing this (which sounds scary and is not actionable for the user) I'd rather write:

```
Upgrading from 0.1 to 0.2 requires [manual migration steps](link to migration guide).
```

This includes all required details, a call to action and is easier on me (I rather look forward, than backwards).

Reply:
Alternatively (but not my favorite) I'd highlight that 0.2 ships with BREAKING CHANGES rather than mentioning "backwards compatibility".

Breaking changes is the term I'm looking for as a developer to understand if I need to do something.

Reply:
```suggestion
Be aware that the update from 0.1 to 0.2 requires manual migration steps as described below.
```